lib/stages.js

```diff
@@ -0,0 +1,21 @@
+const STAGES = [
+  {
+    name: 'review',
+    inferRegex:  /-pr-(\d+)$/
+  },
+  {
+    name: 'development',
+    inferRegex: /-(dev|development)$/
```

Comment:
Good suggestion @markpundsack PRs always welcome and we might even send you a t-shirt ;)